client/src/components/Footer/Footer.css

```
@@ -0,0 +1,56 @@
+footer {
+    background-color: hsl(0, 0%, 70%);
```

we can use the global variables now
```suggestion
    background-color: var(--gray70);
```

---

(don't commit this suggestion using the button above, just change it in your own code)

---

Great, all done.